Resources/Prototypes/Catalog/Fills/Lockers/misc.yml

```diff
@@ -172,6 +172,216 @@
         - id: WeaponSniperMosin
           prob: 0.0010
           orGroup: syndiemaintloot
+        # poster (25%)
+        - id: null
+          prob: 309
```

Comment:
why

Reply:
Groups uses weighted random and spawns only one item from it's entries. This entry is needed to add chance to poster not be spawned and it's 75% of total weight of group.

Reply:
This would generate a "dummy" entity so that isnt really acceptable, `EntitySpawnEntry` is kinda ass to use rn so you will have to assign an individual probability to each entry that combined sums to 100%

Reply:
sure but that could result in posters spawn more than one time

Reply:
now it fails tests

Reply:
I can return orGroup with posters but replace null property with some trash. That should prevent overfill and also will not create "dummy" entities